source/features/faster-pr-diff-options.tsx

```diff
@@ -90,31 +83,43 @@ function init(): false | void {
 		prTitle.title = prTitle.textContent!;
 	}
 
-	// Remove previous options UI
-	const singleCommitUI = select('[data-ga-load^="Diff, view"]');
-
-	if (singleCommitUI) {
-		singleCommitUI.remove();
-		return;
-	}
+	// Only show the native dropdown on medium and small screens #2597
+	select('.js-diff-settings')!.closest('details')!.classList.add('d-lg-none');
+	// Make space for the new button by removing "Changes from" #655
+	(select('.js-commits-filtered') ?? select('[data-hotkey="c"]'))!.firstChild!.remove();
```

Comment:
Why not merge the selectors? `'.js-commits-filtered,[data-hotkey="c"]'`

Reply:
> Why not merge the selectors?

When no commits are selected, the text is directly in the `<summary>` element, but when some commits are selected it's wrapped in a `<span>`:
```html
<summary class="btn-link Link--muted select-menu-button" data-hotkey="c" aria-haspopup="menu" role="button">
      <span class="color-text-link js-commits-filtered">
        Changes from <strong>3 commits</strong>
      </span>
</summary>
```

So `'.js-commits-filtered, [data-hotkey="c"]'` will always select the `<summary>` which is not what we want. There may be a better way to do this though, but I didn't find any.

Reply:
What about `[data-hotkey="c"] strong` + `.previousSibling`?